### DISCUSSION TITLE
Can somebody help me with too many redirects?

### DISCUSSION THREAD

**Discussion by w4sb0y** [nextcloud/all-in-one#3499] [Questions] [answered]
I can't seem to work out what the problem is, but I can't open my nextcloud interface. I installed AIO with docker, and managed to do the first setup. I found this error in my apache container logs: 
`{"level":"error","ts":1696702225.3874214,"logger":"tls.obtain","msg":"will retry","error":"[nextcloud.w4s.hu] Obtain: [nextcloud.w4s.hu] solving challenge: nextcloud.w4s.hu: [nextcloud.w4s.hu] authorization failed: HTTP 400 urn:ietf:params:acme:error:connection - 2606:4700:3031::6815:25eb: Timeout during connect (likely firewall problem) (ca=https://acme-staging-v02.api.letsencrypt.org/directory)","attempt":7,"retrying_in":1200,"elapsed":2446.771298046,"max_duration":2592000} `

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one/discussions/2105
